conda/conda-build-anvil.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
```

Comment:
We request users build the package on a compute node in `conda-build-delta.sh` and `conda-build-greatlakes.sh`. Should we add the relevant `salloc` command as a comment in this script as well?

Reply:
I refactored this into one reusable `build` script with separate job scripts for each platform. Users that build their own packages are free to use whatever resource is most appropriate for building.